yara/yara/core_yara.c

```diff
@@ -95,12 +96,17 @@ static int callback (YR_SCAN_CONTEXT* context, int message, void *msg_data, void
 			YR_MATCH* match;
 			yr_string_matches_foreach(context, string, match)
 			{
+				// Find virtual address if possible
+				RBinSection *s = r_bin_get_section_at (core->bin->cur->o, match->base + match->offset, false);
+				if (s != NULL) {
```

Comment:
!=NULL is implicit, by the r2 coding guidelines we use if (s) { ..

Reply:
what do you think about having an option to decide if you want to VA or PA ? i think both outputs are useful and make sense in this context